### DISCUSSION TITLE
[Art competition] Fuzzer

### DISCUSSION THREAD

**Discussion by babam6a** [prosyslab-classroom/cs348-information-security#405] [Art competition]
![DALL·E 2023-05-23 23 44 07 - the explorer's backside trying to enter a deep dark cave  painted by Claude Monet](https://github.com/prosyslab-classroom/cs348-information-security/assets/65668742/38cf3c28-a0ad-4161-af87-829f882f34d4)

## Short Description
For me, Fuzzer means "the explorer's backside trying to enter a deep dark cave. painted by Claude Monet".

## Long Description (up to 3 sentences)
A cave explorer is a person who moves forward in a cave that cannot be seen an inch ahead, just believeing in his or her senses. It will be same for fuzzing. Fuzzer explores deep, invisible code space just by relying on it's unique algorithm. It may come back without any achievement(cannot find vulnerability), or even not be able to come back(by crash). 
Then, wouldn't Fuzzer feel scary or nervous, like an adventurer trying to get into a deep, dark cave as in picture?

**Comment by KihongHeo** [comment]
Nice try. It would be much nicer if you can express the important concepts in fuzzing we discussed today.
